libs/core/src/components/pds-select/pds-select.scss

```diff
@@ -1,6 +1,7 @@
 :host {
   --box-shadow-focus: 0 0 0 1px #ffffff, 0 0 0 3px var(--pine-color-focus-ring);
   --box-shadow-focus-error: 0 0 0 1px #ffffff, 0 0 0 3px var(--pine-color-focus-ring-danger);
+  --pds-select-min-width: 200px;
```

Comment:
Should we adjust so this just becomes 100% and rely on the parent to control the width?